README.md

```diff
@@ -154,6 +154,64 @@ python -m gcp_variant_transforms.vcf_to_bq \
   --runner DataflowRunner
 ```
 
+### Security and Compliance
+
+You may need to constrain Cloud Dataflow job processing to a specific
+geographic region in support of yourproject’s security and compliance needs.
+For example, the [European Union Data Protection Directive](https://cloud.google.com/security/compliance/eu-data-protection-directive/).
```

Comment:
ditto here. I'd remove reference to the compliance page.

Reply:
Done!